### PR TITLE
create 'intro' lesson

### PR DESCRIPTION
**Pull request by codyloyd**
This PR closes https://github.com/TheOdinProject/top-meta/issues/17 by adding an 'introduction' lesson.

All items from the checklist have been completed.

### CODE REVIEW

File: html_css/v2/foundations/intro-to-html-css.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### Learning Outcomes
+* Get a basic overview of HTML, CSS and how they work together.
+
+### HTML and CSS
+HTML and CSS are two languages that work together to create everything that you see when you look at something on the internet. HTML is the raw data that a webpage is built out of. All the text, links, cards, lists and buttons are created in HTML. CSS is what adds _style_ to those plain elements. HTML is used to put information on a webpage and CSS is used to position that information, give it color, change the font and make it look great!
+
+If you want to get technical, HTML and CSS aren't _programming languages_ because they are only concerned with presenting information. They are not used to program any logic. JavaScript, which you will be learning in the next section, is a programming language because it used to make webpages do things. There is quite a lot you can accomplish with just HTML and CSS however, and you will definitely need them both. The following lessons focus on giving you the tools you need to be successful once you get to the JavaScript content as you continue through our curriculum!
```
Comment (leosoaivan) [inline html_css/v2/foundations/intro-to-html-css.md:10]: `because it's used`

File: html_css/v2/foundations/intro-to-html-css.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+</div>
+
+### Additional Resources
+This section contains helpful links to other content.  It isn't required, so consider it supplemental for if you need to dive deeper into something.
+
+* [This FreeCodeCamp article](https://www.freecodecamp.org/news/html-css-and-javascript-explained-for-beginners/) goes into a little more depth than the assigned one. It covers things we'll be teaching explicitly in later lessons though, so don't worry about memorizing any of the details.
+
+### Knowledge Check
+* What do HTML and CSS stand for?
+* Between HTML and CSS which would you use for putting paragraphs of text on a webpage?
```
Comment (leosoaivan) [inline html_css/v2/foundations/intro-to-html-css.md:24]: `...and CSS, which...`

**Review by leosoaivan — CHANGES_REQUESTED** [review]
LGTM, some grammar needs updating.

File: html_css/v2/foundations/intro-to-html-css.md (hunk 1 above)
Comment (codyloyd) [reply html_css/v2/foundations/intro-to-html-css.md:10]: lol nice.  thanks

File: html_css/v2/foundations/intro-to-html-css.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+### Learning Outcomes
+* Get a basic overview of HTML, CSS and how they work together.
+
+### HTML and CSS
+HTML and CSS are two languages that work together to create everything that you see when you look at something on the internet. HTML is the raw data that a webpage is built out of. All the text, links, cards, lists and buttons are created in HTML. CSS is what adds _style_ to those plain elements. HTML is used to put information on a webpage and CSS is used to position that information, give it color, change the font and make it look great!
+
+If you want to get technical, HTML and CSS aren't _programming languages_ because they are only concerned with presenting information. They are not used to program any logic. JavaScript, which you will be learning in the next section, is a programming language because it's used to make webpages do things. There is quite a lot you can accomplish with just HTML and CSS however, and you will definitely need them both. The following lessons focus on giving you the tools you need to be successful once you get to the JavaScript content as you continue through our curriculum!
+
+### Assignment
+<div class="lesson-content__panel" markdown="1">
```
Comment (thatblindgeye) [inline html_css/v2/foundations/intro-to-html-css.md:13]: The two links in this lesson I think should have better names than "this article".

The first one could be, "Read [HTML vs. CSS vs. Javascript](...) to get a quick overview of the relationship between HTML, CSS, and JavaScript."

The second could be, "[HTML, CSS, and JavaScript Explained for Beginners](...) goes into..."

(first time commenting on a range of lines in a PR)

**Comment by thatblindgeye** [comment]
The two links in this lesson I think should have better names than "this article".

The first one could be, `Read [HTML vs. CSS vs. Javascript](...) to get a quick overview of the relationship between HTML, CSS, and JavaScript.`

The second could be, `[HTML, CSS, and JavaScript Explained for Beginners](...) goes into...`

**Review by KevinMulhern — APPROVED** [review]
LGTM 👍

**Pull request merged**